index.html

```diff
@@ -258,5 +258,22 @@ <h2>Skills</h2>
         </section>
       </section>
     </main>
+    <footer>
+      <section class="contact flex">
+        <p>
+          I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </p>
+        <form class="flex" action="https://formspree.io/f/xdobralq" method="post"
+        >
+          <input name="fname" type="text" required placeholder="Full Name" maxlength="30">
+          <input name="email" type="text" required placeholder="Email address">
+          <textarea name="message" placeholder="Enter text here" maxlength="500"></textarea>

```

Comment:
- Please, kindly make your Textarea field required to prevent submitting the form when its message field is empty.

Reply:
**[OPTIONAL]**

- The footer should only contain the list of social media icons and the contact section should not be wrapped inside of the footer as this was stated in the project requirement, but it is actually not to be added in the footer.  Kindly fix this to improve and enforce best practices in using the html5 semantic tag.

Reply:
all changes made